docs/networks.md

```diff
@@ -0,0 +1,102 @@
+# Networking
+
+The default OpenTofu configurations in the appliance do not provision networks,
+subnets or associated infrastructure such as routers. The requirements are that:
+1. At least one network exists.
+2. The first network defined spans all nodes, referred to as the "access network".
+3. Only one subnet per network is attached to nodes.
+4. At least one network on each node provides outbound internet access (either
```

Comment:
Could we tighten this further to say that the access network *must* provide internet connectivity (as some nodes could only be attached to this network) directly/via proxy, although "extra" networks may or may not provide access to the internet?

Reply:
I think the problem is that which network provides internet might vary across the cluster (have a client in mind!). Although making that work properly for other issues probably also requires https://github.com/stackhpc/ansible-slurm-appliance/pull/539. At this level, I don't think the nodes actually care which interface provides that.